README.md

```diff
@@ -237,13 +240,48 @@ If you encounter errors during install that mention `gyp`, that refers to a Node
 
 #### Develop
 
-To run the app in development mode, start a local web server, and watch for changes do:
+Since the frontend is a static site that just connects to an API for data requests, most frontend development happens independently of server-side changes. The simplest way to do this is to connect the frontend to the test website server. First, copy `frontend/.env.example` into `frontend/.env.development`. This file is already set up to point to the test website server.
```

Comment:
What if we start with definitions of the larger elements and how they are related:
* React App
* Cloud server hosting of the React App
* Data server
* Storybook

Then something like "We use configuration files (frontend/.env.prod, frontend/.env.staging, etc)" to control settings in different environments. These include things like the data server URL and logging settings. These can be overridden."

(Maybe this is in a ###Overview section between ###Install and ###Develop)

And then have the sections be of the format:
1. Running just the React App
2. Running the Frontend Server locally
3. Running the Frontend Server with Docker locally
4. Running frontend in your own GCP project
5. Running Storybook (moving it from below up here)

And for each section
-> When/why you would want to do this
-> How to do it (commands)
-> Default behavior, and and any available overrides

What do you think?

Reply:
Did my best to match that structure, with some minor changes. Let me know what you think and we can iterate going forward!